xml/CRSChannelCode.xml

```diff
@@ -9,2499 +9,4945 @@
             </name>
+            <description>
+                <narrative/>
+                <narrative xml:lang="fr"/>
+            </description>
+            <category>32000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
-            <code>21017</code>
-            <name>
-                <narrative>International Centre for Trade and Sustainable Development</narrative>
-                <narrative xml:lang="fr">Centre international de commerce et de développement durable</narrative>
-            </name>
-        </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
+        <codelist-item status="active" activation-date="2010-01-01">
             <code>31003</code>
             <name>
                 <narrative>International Land Coalition</narrative>
                 <narrative xml:lang="fr">Coalition internationale pour l'accès à la terre</narrative>
             </name>
+            <description>
+                <narrative/>
+                <narrative xml:lang="fr"/>
+            </description>
+            <category>32000</category>
+        </codelist-item>
+        <codelist-item status="active" activation-date="2010-01-01">
+            <code>31004</code>
+            <name>
+                <narrative>Extractive Industries Transparency Initiative International Secretariat</narrative>
+                <narrative xml:lang="fr">Secrétariat de l'Initiative pour la transparence dans les industries extractives</narrative>
+            </name>
+            <description>
+                <narrative/>
+                <narrative xml:lang="fr"/>
+            </description>
+            <category>32000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
+        <codelist-item status="active" activation-date="2011-01-01">
             <code>31005</code>
             <name>
                 <narrative>Parliamentary Network on the World Bank</narrative>
                 <narrative xml:lang="fr">Réseau parlementaire sur la Banque mondaile</narrative>
             </name>
+            <description>
+                <narrative/>
+                <narrative xml:lang="fr"/>
+            </description>
+            <category>32000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
-            <code>40000</code>
+        <codelist-item status="active" activation-date="2017-01-01">
+            <code>31006</code>
             <name>
-                <narrative>MULTILATERAL ORGANISATIONS</narrative>
-                <narrative xml:lang="fr">ORGANISATIONS MULTILATÉRALES</narrative>
+                <narrative>Coalition for Epidemic Preparedness Innovations</narrative>
+                <narrative xml:lang="fr">Coalition pour les innovations en matière de préparation</narrative>
```

Comment:
I think this should be: Coalition pour les innovations en matière de préparation **aux épidémies**.

Reply:
(looks like it’s a bug in the source, though).